changelog.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## Version 3.1.2 [03-April-2018]
```

Comment:
https://github.com/causecode/grails-file-uploader/pull/30/files#diff-42ba1d994f4fa8c6ad17a7efae7936ccR1

Add changelog for 3.1.0

Reply:
Ok